gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/remote/DriverRemoteConnection.java

```diff
@@ -210,7 +214,20 @@ public static DriverRemoteConnection using(final Configuration conf) {
     @Override
     public <E> CompletableFuture<RemoteTraversal<?, E>> submitAsync(final Bytecode bytecode) throws RemoteConnectionException {
         try {
-            return client.submitAsync(bytecode).thenApply(rs -> new DriverRemoteTraversal<>(rs, client, attachElements, conf));
+            final Iterator<OptionsStrategy> itty = IteratorUtils.map(
+                    IteratorUtils.filter(bytecode.getSourceInstructions().iterator(),
```

Comment:
I don't know that we'd expose `RequestOptions` as the return result from a utility method because that is a class specific to the TP driver. However, I do think it would be good to extract the code that finds the `OptionsStrategy` list from `Bytecode`.